content/purgatorio/collect-and-prepare-data/data-visualization.md

```diff
@@ -52,6 +52,12 @@ I assume you know [basic Python](https://github.com/clone95/Virgilio/blob/master
 
 Each content listed here **is not** tool-specific (apart from "tools", did you ever imagine that?).
 
+Data visualisation is NOT a step, in fact it is a part of every data and machine learning related step, we start with it, we do it at every step, before and after and we finish with it. See other visualisation steps across the Data science process (pipelines):
+- [Visualisations during Data Collection](https://github.com/virgili0/Virgilio/blob/dev/content/purgatorio/collect-and-prepare-data/data-collection.md#visualization)
+- [Visualisations during Data Preparation](https://github.com/virgili0/Virgilio/blob/dev/content/purgatorio/collect-and-prepare-data/data-preparation.md#visualization)
+- [Visualisations during Exploratory Data Analysis](https://github.com/virgili0/Virgilio/blob/dev/content/purgatorio/collect-and-prepare-data/exploratory-data-analysis.md#visualization)
+- [Visualisations during Feature Engineering](https://github.com/virgili0/Virgilio/blob/dev/content/purgatorio/collect-and-prepare-data/feature-engineering.md#visualization)
+
```

Comment:
To put reference for a specific section of the "other" guide with it's visualization section?
Did I understand right?
If it's the case, it's a very good idea!

Reply:
Yes - each section will have its own Viz section and then this one points to all of them
As Visualisation isn't a separate task, its part and parcel of each step in the process sometimes before and after each process!

So much to write about these topics!

Reply:
This PR is still in Draft stage but it's okay to glance it - I have to flesh them out